pkg/collector/horizontalpodautoscaler.go

```diff
@@ -37,7 +37,12 @@ func HorizontalPodAutoscaler(cfg config.Config, logger logr.Logger, otelcol v1al
 	labels["app.kubernetes.io/name"] = naming.Collector(otelcol)
 
 	annotations := Annotations(otelcol)
-	cpuTarget := defaultCPUTarget
+	var cpuTarget int32
+	if otelcol.Spec.TargetCPUUtilization != nil {
+		cpuTarget = *otelcol.Spec.TargetCPUUtilization
+	} else {
+		cpuTarget = defaultCPUTarget
```

Comment:
the default value of  `otelcol.Spec.TargetCPUUtilization should be set in the defaulting webhook.

Reply:
Thanks for the review! moved the setting of default to opentelemetrycollector_webhook.go